src/core/entities/MRImageEntity.js

```diff
@@ -75,7 +75,7 @@ export class MRImageEntity extends MRMediaEntity {
         // passed through MRMediaEntity directly is undefined since it is not
         // a direct element for users. So we do the if-check here and then
         // follow the same as the parent's functionality.
-        if (mutation.type != 'attributes' && mutation.attributeName == 'src') {
+        if (mutation.type == 'attributes' && mutation.attributeName == 'src') {
```

Comment:
not sure i understand the reasoning for this swap - was this originally a typo?

Reply:
it was and I think through cut/paste it propagated. `src` is of type `attributes`.

Reply:
gotcha, that copypasta 'shakes fist at cloud'